<!DOCTYPE html>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"><link type="text/css" rel="stylesheet" href="resources/sheet.css" >
<style>
  table{
    border: 2px solid black;
    border-collapse: collapse;
    margin: 25px 0;
    font-size: 0.9em;
    font-family: sans-serif;
    min-width: 400px;
    
    width: 80%;    
    box-shadow: 0 0 20px rgba(31,05,62, 0.15);
	}
   th, td {
    border: 1px solid black;
   }
   body {
    font: normal 14px sans-serif;
   }

</style>


<body>
<h1>FAIRplus FAIR indicators</h1>
<p><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/FAIR%2BIndicators.md">version: v0.01</a></p>
</blockquote>
<p>FAIRplus indicators are designed for measuring data sets compliance 
to Data Usage Areas. One indicator might support more than one Data 
Usage Area. Indicators are grouped according to the ISA framework.</p>
<p><a target="_blank" rel="noopener noreferrer" href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/img/isa-structure.png"><img src="/fairification-results/img/isa-structure.png" alt="ISA" style="max-width:100%;"></a></p>
<p>Image Reference : <a href="https://www.researchgate.net/publication/333163209_Design_and_Development_of_a_Phenotypic_Data_Model_PDM" rel="nofollow">Khan</a></p>
<h2><a id="user-content-index" class="anchor" aria-hidden="true" href="#index"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Index</h2>
<table>
<thead>
<tr>
<th>ID</th>
<th>Indicators</th>
</tr>
</thead>
<tbody>
<tr>
<td>F+S01</td>
<td>Study level documentation is available in a human readable format.</td>
</tr>
<tr>
<td>F+S02</td>
<td>Data is reported by following community specific minimum information guidelines</td>
</tr>
<tr>
<td>F+S03</td>
<td>Metadata documents and provides references about all data biological data types and formats in data is expressed.</td>
</tr>
<tr>
<td>F+S04</td>
<td>Relationships between different data sets in a study is well defined.</td>
</tr>
<tr>
<td>F+S05</td>
<td>A versioning policy is applied to uniquely identify a particular 
form of a dataset from an earlier form or other forms of itself.</td>
</tr>
<tr>
<td>F+S06</td>
<td>Share not only derived and publication related data but data 
generated in early phases of research data workflow such as primary data
 and analyzed data.</td>
</tr>
<tr>
<td>F+S07</td>
<td>Negative results are shared.</td>
</tr>
<tr>
<td>F+S08</td>
<td>The study is described with metadata including context, samples and 
data acquisition, methods for analyzing and processing data, quality 
control, and restriction for reuse.</td>
</tr>
<tr>
<td>F+S08a</td>
<td>Metadata includes information about the study design, protocols and data collection methods.</td>
</tr>
<tr>
<td>F+S08b</td>
<td>Metadata includes explicit references to research resources such as samples, cell lines</td>
</tr>
<tr>
<td>F+S08c</td>
<td>Metadata contains information about data processing methods, data analysis and quality assurance metrics.</td>
</tr>
<tr>
<td>F+S08d</td>
<td>Metadata includes information about data ownership, license and reuse constraints for sensitive data.</td>
</tr>
<tr>
<td>F+A01</td>
<td>Data is organized and documented in a human understandable way</td>
</tr>
<tr>
<td>F+A02</td>
<td>Data is encoded in a community specific exchange standard.</td>
</tr>
<tr>
<td>F+A03</td>
<td>A machine and human readable formal description of the structure of data is available including types, properties.</td>
</tr>
<tr>
<td>F+A04</td>
<td>Data is structured by following a life sciences domain model, core 
classes and their semantic relations refers to a common data model.</td>
</tr>
<tr>
<td>F+A05</td>
<td>Data is described with terminology standards.</td>
</tr>
<tr>
<td>F+A06</td>
<td>Core data classes (important data elements) follows a common master and reference data entity.</td>
</tr>
</tbody>
</table>
<h3><a id="user-content-study-level" class="anchor" aria-hidden="true" href="#study-level"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Study Level</h3>
<h4><a id="user-content--fs01---study-level-documentation-is-available-in-a-human-readable-format" class="anchor" aria-hidden="true" href="#-fs01---study-level-documentation-is-available-in-a-human-readable-format"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s01"> F+S01 </a> : Study level documentation is available in a human readable format.</h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Study-level documentation provides high-level information on the 
research context and design, the data collection methods used, any data 
preparations and manipulations and summaries of findings based on the 
data. Examples and a suggested list of coverage can be found at <a href="https://www.ukdataservice.ac.uk/manage-data/document/study-level.aspx" rel="nofollow">UK Data Services</a>. <br><br>This
 indicator does not require any machine interpretable content, or 
semantic annotation of information with common vocabularies. It 
measures, availability of information to help data users to make 
informed use of the data. Publicly accessible web pages, pdf documents 
are acceptable formats.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs02---data-is-reported-by-following-community-specific-minimum-information-guidelines" class="anchor" aria-hidden="true" href="#-fs02---data-is-reported-by-following-community-specific-minimum-information-guidelines"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s02"> F+S02 </a> : <strong>Data is reported by following community specific minimum information guidelines</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>A reporting standard ensures recording the information (metadata) 
required to unambiguously communicate experimental designs, treatments 
and analyses, to con-textualize the data generated. Such standards are 
also known as data content or minimum information standards (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/" rel="nofollow">Chervitz, et all.</a>). See examples: reporting standards for<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/table/T8/?report=objectonly" rel="nofollow"> health care</a>, for <a href="https://www.biorxiv.org/content/biorxiv/early/2017/07/24/167619/T4/graphic-10.large.jpg" rel="nofollow">life sciences</a> data, collection of <a href="https://fairsharing.org/standards/?q=minimum+information&amp;selected_facets=type_exact:reporting%20guideline" rel="nofollow">FAIRsharing </a></td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Reproducibility">Reproducibility</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs03----metadata-documents-and-provides-references-about-all-data-biological-data-types-and-formats-in-data-is-expressed" class="anchor" aria-hidden="true" href="#-fs03----metadata-documents-and-provides-references-about-all-data-biological-data-types-and-formats-in-data-is-expressed"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s03"> F+S03 </a> :  <strong>Metadata documents and provides references about all data biological data types and formats in data is expressed.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Biological and biomedical research has been considered an especially
 challenging research field in this regard, as data types are extremely 
heterogeneous and not all have defined data standards (<a href="https://www.biorxiv.org/content/10.1101/167619v1.full" rel="nofollow">Griffin, et. all</a>).
 Metadata should capture all data types and format names in a study, if 
possible provide a reference or URL for format specification, if not 
possible have a description.<br><br><a href="https://www.biorxiv.org/highwire/markup/341342/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" rel="nofollow">Example </a>list of common standard data formats for omics data.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Interpretability">Interpretability</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs04--relationships-between-different-data-sets-in-a-study-is-well-defined" class="anchor" aria-hidden="true" href="#-fs04--relationships-between-different-data-sets-in-a-study-is-well-defined"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s04"> F+S04</a> : <strong>Relationships between different data sets in a study is well defined.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>In a study there are multiple data sets which are used as input and 
produced as an output. When a data is FAIRified, it is important to 
understand which data files are generated from the analysis of which 
other data sets, or sample data. For example the EMBL-EBI <a href="https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html" rel="nofollow">SDRF </a>(Sample
 and Data Relationship Format) describes the sample characteristics and 
the relationship between samples, arrays, data files. Some communities 
such as proteomics have adopted these file formats to their needs, see <a href="https://github.com/bigbio/proteomics-metadata-standard">SDRF for Proteomics</a> . Furthermore, <a href="https://isa-tools.org/format/specification.html" rel="nofollow">ISA</a> allows multiomics support and is used by <a href="https://www.ebi.ac.uk/metabolights/index" rel="nofollow">EMBL-EBI Metabolights</a> <br><br>The relationships can be also expressed as naming a convention. A good example is the <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/" rel="nofollow">TCGA </a>data
 set, where the barcode contains semantic information about the relation
 of data with other data (such as participant, sample, analyte, plate, 
sample). If there is a naming convention used, instead of a defined 
format standard, this convention should be clearly documented together 
with <a href="https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables" rel="nofollow">code tables</a>.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Interpretability">Interpretability</a>  + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs05---a-versioning-policy-is-applied-to-uniquely-identify-a-particular-form-of-a-dataset-from-an-earlier-form-or-other-forms-of-itself" class="anchor" aria-hidden="true" href="#-fs05---a-versioning-policy-is-applied-to-uniquely-identify-a-particular-form-of-a-dataset-from-an-earlier-form-or-other-forms-of-itself"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s05"> F+S05 </a> : <strong>A
 versioning policy is applied to uniquely identify a particular form of a
 dataset from an earlier form or other forms of itself.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Versioning is tracking the changes made in data by saving new copies
 of data files with indicators of the changes made. A new version is 
created when there is a change in the structure, contents, or condition 
of the resource. In the case of research data, a new version of a 
dataset may be created when an existing dataset is reprocessed, 
corrected or appended with additional data. Versioned data are required 
to cite and identify the exact dataset used as a research input in order
 to support research reproducibility and trustworthiness (<a href="https://www.ands.org.au/working-with-data/data-management/data-versioning" rel="nofollow">ANDS</a>). <br><br>Best
 practices of versioning should include a numbering system, information 
about the status of the file, and what changes are made (see YALE 
Research Data Management for the Health Sciences: <a href="https://guides.library.yale.edu/rdm_healthsci/versioning#:~:text=Versioning" rel="nofollow">File Versioning </a>
 involves tracking the changes,access older copies of files.)) . PIDs 
can be assigned for versions, however there is no convention for that 
(see <a href="https://www.ands.org.au/working-with-data/data-management/data-versioning" rel="nofollow">ANDS</a> : DOIs for versioned data). Some simple versioning solutions can be adopted (see <a href="https://library.stanford.edu/research/data-management-services/data-best-practices/data-versioning" rel="nofollow">Stanford libraries</a>, Uni. <a href="https://data.library.virginia.edu/data-management/plan/files/" rel="nofollow">Virginia Library</a>).</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Reproducibility">Reproducibility</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs06---share-not-only-derived-and-publication-related-data-but-data-generated-in-early-phases-of-research-data-workflow-such-as-primary-data-and-analyzed-data" class="anchor" aria-hidden="true" href="#-fs06---share-not-only-derived-and-publication-related-data-but-data-generated-in-early-phases-of-research-data-workflow-such-as-primary-data-and-analyzed-data"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s06"> F+S06 </a> : <strong>Share
 not only derived and publication related data but data generated in 
early phases of research data workflow such as primary data and analyzed
 data.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Life science experiments comprise different samples; based on 
experiment and sample, several measurements are performed. After 
quantification there is a step of data processing and analysis which 
results in life science research (<a href="https://www.researchgate.net/publication/51724887_A_case_study_for_efficient_management_of_high_throughput_primary_lab_data/figures" rel="nofollow">Colmsee</a>
 et.all). In most cases, data is shared with publications as 
supplementary files, or in data repositories which are referenced by 
articles. However raw data and primary data resides in private storages (<a href="https://www.researchgate.net/publication/263355191_EDAL_-_a_framework_to_store_share_and_publish_research_data/figures" rel="nofollow">Arend</a>,
 et. all). There are great benefits of sharing primary data, including 
to make meaningful comparisons between results, to answer ‘what if’ 
questions and to carry out pilot studies without repeating experiments (<a href="https://www.nature.com/articles/nn0900_863" rel="nofollow">Koslow</a> ). <br><br>This indicator measures availability of any primary data beyond the derived data or aggregated analyzed data.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs07---negative-results-are-shared" class="anchor" aria-hidden="true" href="#-fs07---negative-results-are-shared"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s07"> F+S07 </a> : <strong>Negative results are shared.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Negative results are the outcomes which do not support study 
hypotheses. Researchers are rewarded more for publishing novel findings,
 and not for publishing negative results. However, sharing negative 
results could reduce efforts and avoid repeating work that may be 
difficult to replicate (<a href="https://www.nature.com/articles/d42473-019-00004-y#ref-CR15" rel="nofollow">ATCC</a>).Negative result also can be reported in publications (e.g. <a href="https://www.negative-results.org/" rel="nofollow">Negative Results Journal</a>). <br><br>This indicator measures the availability of negative results sets in the shared data content.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs08---the-study-is-described-with-metadata-including-context-samples-and-data-acquisition-methods-for-analyzing-and-processing-data-quality-control-and-restriction-for-reuse" class="anchor" aria-hidden="true" href="#-fs08---the-study-is-described-with-metadata-including-context-samples-and-data-acquisition-methods-for-analyzing-and-processing-data-quality-control-and-restriction-for-reuse"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s08"> F+S08 </a> : <strong>The
 study is described with metadata including context, samples and data 
acquisition, methods for analyzing and processing data, quality control,
 and restriction for reuse.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>This indicator can be evaluated in two phases: 1) providing a 
structured metadata with domain conventions; 2) providing machine 
readable metadata by using any common vocabularies. <br><br>This indicator is divided into a set of indicators from F+S08a to F+S08d.</td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs08a---metadata-includes-information-about-the-study-design-protocols-and-data-collection-methods" class="anchor" aria-hidden="true" href="#-fs08a---metadata-includes-information-about-the-study-design-protocols-and-data-collection-methods"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s08a"> F+S08a </a> : <strong>Metadata includes information about the study design, protocols and data collection methods.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Example metadata: study / experiment design, trial protocol, data 
acquisition methods, experiment methods, data processing methods .<br>See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3903324/table/T1/" rel="nofollow">multi omics metadata</a> check list, <a href="https://www.ncbi.nlm.nih.gov/books/NBK286004/box/box_4-1/?report=objectonly" rel="nofollow">clinical trial metadata</a></td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Reproducibility">Reproducibility</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs08b--metadata-includes-explicit-references-to-research-resources-such-as-samples-cell-lines" class="anchor" aria-hidden="true" href="#-fs08b--metadata-includes-explicit-references-to-research-resources-such-as-samples-cell-lines"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s08b"> F+S08b</a> : <strong>Metadata includes explicit references to research resources such as samples, cell lines</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Provide information about key research materials such as antibodies,
 cell lines, and organisms. Preferable with identifiers, see <a href="https://www.rrids.org/" rel="nofollow">RRID </a>portal.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Reproducibility">Reproducibility</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs08c--metadata-contains-information-about-data-processing-methods-data-analysis-and-quality-assurance-metrics" class="anchor" aria-hidden="true" href="#-fs08c--metadata-contains-information-about-data-processing-methods-data-analysis-and-quality-assurance-metrics"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s08c"> F+S08c</a> : <strong>Metadata contains information about data processing methods, data analysis and quality assurance metrics.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>See specific examples for quality requirements for <a href="https://link.springer.com/chapter/10.1007/164_2019_284" rel="nofollow">In Vitro research</a>, such as chemical probes, cell line authentication, antibody validation.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Reproducibility">Reproducibility</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fs08d--metadata-includes-information-about-data-ownership-license-and-reuse-constraints-for-sensitive-data" class="anchor" aria-hidden="true" href="#-fs08d--metadata-includes-information-about-data-ownership-license-and-reuse-constraints-for-sensitive-data"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+s08d"> F+S08d </a> :<strong>Metadata includes information about data ownership, license and reuse constraints for sensitive data.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Considering the sensitive nature of life science data, beside data 
reuse license, consent, and if exist any other constraints should be 
documented.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Repurposing">Repurposing</a></td>
</tr>
</tbody>
</table>
<hr>
<h3><a id="user-content-assays--data-set-level" class="anchor" aria-hidden="true" href="#assays--data-set-level"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>Assays / Data Set Level:</h3>
<h4><a id="user-content--fa01---data-is-organized-and-documented-in-a-human-understandable-way" class="anchor" aria-hidden="true" href="#-fa01---data-is-organized-and-documented-in-a-human-understandable-way"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a01"> F+A01 </a> : <strong>Data is organized and documented in a human understandable way</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Data-level, or object-level, documentation provides information at 
the level of variables in a database or individual objects such as 
images. Data-level information can be embedded in data files, such as 
variable, value and code labels in an SPSS file or headers in a 
document. Examples for quantitative, qualitative and secondary source 
documentation can be found at <a href="https://www.ukdataservice.ac.uk/manage-data/document/study-level.aspx" rel="nofollow">UK Data Services</a>.<br><br>This
 indicator does not require any machine readable content, or semantic 
annotation of information with common vocabularies. It measures how 
easily data can be explored and understood by humans who are familiar 
with the domain.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Interpretability">Interpretability</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fa02---data-is-encoded-in-a-community-specific-exchange-standard" class="anchor" aria-hidden="true" href="#-fa02---data-is-encoded-in-a-community-specific-exchange-standard"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a02"> F+A02 </a> : <strong>Data is encoded in a community specific exchange standard.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>A data exchange standard defined the encoding format of data. A data
 exchange standard delineates what data types can be encoded and the 
particular way they should be encoded (e.g., tab-delimited columns, XML,
 binary, etc. They facilitate the exchange of information between 
researchers and organizations, and between software programs or 
information storage systems. ). They provide syntax standards but do not
 specify what the document should contain in order to be considered 
complete (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/" rel="nofollow">Chervitz, et all.</a>).<br><br> See examples for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/table/T4/" rel="nofollow">omic data</a>, for microarray-based <a href="https://www.researchgate.net/publication/257204855_A_sea_of_standards_for_omics_data_Sink_or_swim/figures" rel="nofollow">transcriptomics </a>, for <a href="https://en.wikipedia.org/wiki/Clinical_Data_Interchange_Standards_Consortium#Individual_standards" rel="nofollow">clinical data</a>.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Integration">Integration</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fa03---a-machine-and-human-readable-formal-description-of-the-structure-of-data-is-available-including-types-properties" class="anchor" aria-hidden="true" href="#-fa03---a-machine-and-human-readable-formal-description-of-the-structure-of-data-is-available-including-types-properties"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a03"> F+A03 </a> : <strong>A machine and human readable formal description of the structure of data is available including types, properties.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>A schema describes the structure of the data. Special schemes have 
meanings associated with databases, such as community agreed profiles. A
 schema consists of a key dimension and its properties, expected types, 
constraints, cardinalities and associated controlled vocabularies 
(preferably refers to existing ontologies). Schemas and profiles can be 
registered and reused, for examples <a href="https://fairsharing.org/standards/" rel="nofollow">FAIRsharing Standards</a> and specific examples such as <a href="https://schema.org/docs/schemas.html" rel="nofollow">Schema.org</a>, <a href="https://bioschemas.org/profiles/" rel="nofollow">Bioschemas</a>, or <a href="https://www.hl7.org/fhir/resourcelist.html" rel="nofollow">HL7 resources</a> in the context of health data records.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Integration">Integration</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content--fa04---data-is-structured-by-following-a-life-sciences-domain-model-core-classes-and-their-semantic-relations-refers-to-a-common-data-model" class="anchor" aria-hidden="true" href="#-fa04---data-is-structured-by-following-a-life-sciences-domain-model-core-classes-and-their-semantic-relations-refers-to-a-common-data-model"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a04"> F+A04 </a> : <strong>Data
 is structured by following a life sciences domain model, core classes 
and their semantic relations refers to a common data model.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Meaningful exchange of information is a fundamental challenge in 
life sciences research. A domain model A domain is an abstract, 
implementation-independent representation of the grammar, or semantics, 
of a domain (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486731/" rel="nofollow">Freimuth, et. all)</a>
 . Domain models define core classes and the semantic relationships 
between them, as well as providing unambiguous definitions for concepts 
required to describe life sciences research. Examples of life sciences 
common data models are <a href="https://www.ohdsi.org/data-standardization/the-common-data-model/" rel="nofollow">OMOP</a>, <a href="https://bridgmodel.nci.nih.gov/high-level-concept" rel="nofollow">BRIDG</a>, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486731/" rel="nofollow">Lifesciences DAM</a>, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1262694/figure/F1/" rel="nofollow">functional genomics</a> data modelling.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Interpretability">Interpretability</a>  + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Integration">Integration</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content-fa05---data-is-described-with-terminology-standards" class="anchor" aria-hidden="true" href="#fa05---data-is-described-with-terminology-standards"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a05">F+A05 </a> : Data is described with terminology standards.</h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Terminology standards is typically defined by the use cases and 
provides control vocabularies to support and competency questions it is 
designed to answer. In life sciences domain ontologies are common ways 
to encode terminology standards (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/" rel="nofollow">Chervitz, et all.</a>).
 Terminology standards add an interpretive layer to the data by defining
 the concepts or terms in a domain, and in some cases the relationships 
between them (<a href="https://doi.org/10.1136/amiajnl-2013-002066" rel="nofollow">Tenenbaum et. all</a>). See an example list for <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4152841/table/T5/" rel="nofollow">terminology standards</a>. For a complete listing see the <a href="http://www.obofoundry.org/" rel="nofollow">OBO </a>Foundry.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Interpretability">Interpretability</a> + <a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Integration">Integration</a></td>
</tr>
</tbody>
</table>
<h4><a id="user-content-fa06----core-data-classes-important-data-elements-follows-a-common-master-and-reference-data-entity" class="anchor" aria-hidden="true" href="#fa06----core-data-classes-important-data-elements-follows-a-common-master-and-reference-data-entity"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a><a name="user-content-f+a06">F+A06  </a> : <strong>Core data classes (important data elements) follows a common master and reference data entity.</strong></h4>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>Description</td>
<td>Master data is defined as core business objects used in different 
applications across an organization along with their associated 
metadata, definitions and taxonomies. Reference data is used to 
characterize or classify other data such as codes and description 
tables. Master and Reference data lowers cost and complexity through use
 of standards, common data models, and integration patterns. Sharing 
master data within a community or in organization reduces variability 
caused by multiple studies producing the same type of data, but in 
isolation, then inconsistencies in data structure and data values 
between the systems occurs (DAMA). <br>In the life science domain, 
master data entities examples can be a study, a subject, a file, a 
chemical compound, or an observation. Common data models provide core 
classes which can be useful for creating master data elements (see<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486731/" rel="nofollow"> LS DAM </a>Experiment,
 Molecular Biology, Molecular Databases core classes). However master 
data should be described, are assigned an unique identifier, and 
registered on a searchable source. An example to reference data from the
 health domain can be the <a href="https://simplifier.net/guide/ProfilingAcademy/Terminology" rel="nofollow">value set</a> of the HL7.<br><br>Precondition
 of measuring this indicator is the existence of a master and reference 
data in a related research community, an organization, or a repository.</td>
</tr>
<tr>
<td>Related DU Area</td>
<td><a href="https://github.com/FAIRplus/CMM/blob/v0.1/docs/Data_Usage_Areas.md#Integration">Integration</a></td>
</tr>
</tbody>
</table>
</article>
  </div>

    </div>
</body>


<div aria-live="polite" class="sr-only"></div><tmpopup style="top: 986px; left: 452.233px;"><tmpopupcolor style="background: rgb(221, 153, 255) none repeat scroll 0% 0%;" id="tmpopupcolor--1"></tmpopupcolor><tmpopupcolor style="background: rgb(102, 187, 255) none repeat scroll 0% 0%;" id="tmpopupcolor--2"></tmpopupcolor><tmpopupcolor style="background: rgb(85, 255, 85) none repeat scroll 0% 0%;" id="tmpopupcolor--3"></tmpopupcolor><tmpopupcolor style="background: rgb(255, 102, 102) none repeat scroll 0% 0%;" id="tmpopupcolor--4"></tmpopupcolor><tmpopupcolor style="background: rgb(255, 204, 0) none repeat scroll 0% 0%;" id="tmpopupcolor--5"></tmpopupcolor><tmpopupcolor style="background: rgb(255, 238, 0) none repeat scroll 0% 0%;" id="tmpopupcolor--m"></tmpopupcolor></tmpopup></body></html>